README.md

```diff
@@ -16,6 +16,13 @@ To view or contribute Charts, head over to the [charts repo](https://github.com/
 
 ![Deis Graphic](https://s3-us-west-2.amazonaws.com/get-deis/deis-graphic-small.png)
 
+**If you are a Helm 0.1 user** you will need to do an extra step when
+you upgrade to Helm 0.2. We changed our GitHub org from `deis` to
+`helm`, which means the new default charts repository is now
+`github.com/helm/charts`. To fix quickly, simply run `helm repo rm
+charts` followed by `helm repo add charts
+https://github.com/helm/charts`.
+
```

Comment:
Does that feel like it "breaks out of" the little notification area? I left it inline because I only wanted people who are upgrading from 0.1 to do it. I was afraid calling it out in a code block would result in confusion from people who were just skimming the docs.